toolchain/lowering/lowering_handle.cpp

```diff
@@ -123,9 +122,16 @@ auto LoweringHandleCall(LoweringFunctionContext& context,
   for (auto ref_id : context.semantics_ir().GetNodeBlock(refs_id)) {
     args.push_back(context.GetLocalLoaded(ref_id));
   }
-  auto* value =
-      context.builder().CreateCall(function, args, function->getName());
-  context.SetLocal(node_id, value);
+  if (function->getReturnType()->isVoidTy()) {
+    context.builder().CreateCall(function, args);
+    // TODO call GetType with empty tuple type.
```

Comment:
```suggestion
    // TODO: Call GetType with an empty tuple type.
```

Just fixing towards standard TODO comment structure here.